cli/azd/pkg/commands/pipeline/cmd_util_duplicate.go

```diff
@@ -0,0 +1,302 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package pipeline
```

Comment:
Why is the `duplicate` util module here vs reusing the original one?

Reply:
My recommendation would be to move the pipeline manager and provider implementations into a `pkg/pipelines` package and reference them from your pipeline command.  This will allow use of the utils and other common code.

Reply:
I really think we should figure out a way to remove this duplication - even if it means keeping a little logic in `cmd/pipeline.go` for now.  I think at a high level the code in `cmd/pipeline.go` should do things like load the environment, load the project and then construct a manager with these and call `configure` or something along these lines.